betterproto/tests/inputs/namespace_builtin_types/namespace_builtin_types.json

```diff
@@ -0,0 +1,16 @@
+{
+  "int": "value-for-int",
```

Comment:
what's with these values?

Reply:
nothing much :-P

but the fields neem some values.. i don't like `{"int": "int"}` very much, because when you see errors it becomes confusing if its the value or the key. same, if all fields have the same value like "foo", you can't see which field crashed if only the value was printed in the error.
so i did it like this.

i could also make all of them ints with unique values like 1 2 3...